.gitattributes

```diff
@@ -17,7 +17,7 @@ bld.bat text eol=crlf
 .gitattributes linguist-generated=true
 .gitignore linguist-generated=true
 .travis.yml linguist-generated=true
-.scripts linguist-generated=true
+.scripts/* linguist-generated=true
```

Comment:
I think `.scripts/` is the whole directory. The trailing `*` is not needed and might not match dot-files. At least, that's how `.gitignore` files work.

Reply:
All files outside of `recipe/` are auto-generated/-updated.